src/algorithms/policy_gradient/sac.jl

```diff
@@ -23,6 +23,9 @@ mutable struct SACPolicy{
     update_every::Int
     step::Int
     rng::R
+    # Logging
+    reward_term::Float32
+    entropy_term::Float32
```

Comment:
Added terms for the reward and entropy part of the policy loss to make it easier to track and log those. Thought it could be good to have that access when choosing the weighting factor alpha.